src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/scheduler/SendOutstandingOrdersNeedReviewNotificationTask.java

```diff
@@ -0,0 +1,159 @@
+package uk.gov.hmcts.reform.finrem.caseorchestration.scheduler;
+
+import lombok.extern.slf4j.Slf4j;
+import org.apache.lucene.search.join.ScoreMode;
+import org.elasticsearch.index.query.BoolQueryBuilder;
+import org.elasticsearch.index.query.QueryBuilders;
+import org.elasticsearch.index.query.TermQueryBuilder;
+import org.elasticsearch.search.builder.SearchSourceBuilder;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.ccd.client.model.SearchResult;
+import uk.gov.hmcts.reform.finrem.caseorchestration.mapper.FinremCaseDetailsMapper;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.CaseType;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.FinremCaseDetails;
+import uk.gov.hmcts.reform.finrem.caseorchestration.service.CcdService;
+import uk.gov.hmcts.reform.finrem.caseorchestration.service.DraftOrderService;
+import uk.gov.hmcts.reform.finrem.caseorchestration.service.NotificationService;
+import uk.gov.hmcts.reform.finrem.caseorchestration.service.SystemUserService;
+import uk.gov.hmcts.reform.finrem.caseorchestration.utils.csv.CaseReference;
+
+import java.util.List;
+
+import static uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.draftorders.review.OrderStatus.TO_BE_REVIEWED;
+
+/**
+ * Scheduled task to send notification about outstanding orders need review (contested).
+ * To enable the task to execute set environment variables:
+ * <ul>
+ *     <li>CRON_SEND_OUTSTANDING_ORDERS_NEED_REVIEW_ENABLED=true</li>
+ *     <li>TASK_NAME=SendOutstandingOrdersNeedReviewNotificationTask</li>
+ *     <li>CRON_SEND_OUTSTANDING_ORDERS_NEED_REVIEW_BATCH_SIZE=number of cases to search for</li>
+ *     <li>CRON_SEND_OUTSTANDING_ORDERS_NEED_REVIEW_DAYS_SINCE_ORDER_UPLOAD=The number of days
+ *     after the agreed draft order or pension sharing annex upload before sending a notification to review outstanding orders.</li>
+ * </ul>
+ */
+@Component
+@Slf4j
+public class SendOutstandingOrdersNeedReviewNotificationTask extends BaseTask {
+
+    private static final String TASK_NAME = "SendOutstandingOrdersNeedReviewNotificationTask";
+    private static final String SUMMARY = "DFR-3329";
```

Comment:
I think it would be better to make this more meaningful.
"Draft order review overdue notification sent"

It would just help when we're doing support and looking at case event records